master-firmware/src/config.c

```diff
@@ -306,6 +316,14 @@ void config_init(void)
     parameter_scalar_declare_with_default(&main_arm.control.y.ki, &main_arm.control.y.ns, "ki", 0);
     parameter_scalar_declare_with_default(&main_arm.control.y.kd, &main_arm.control.y.ns, "kd", 0);
     parameter_scalar_declare_with_default(&main_arm.control.y.ilimit, &main_arm.control.y.ns, "i_limit", 0);
+
+    parameter_namespace_declare(&screen.ns, &master_config, "screen");
+    parameter_scalar_declare_with_default(&screen.ax, &screen.ns, "ax", 0);
+    parameter_scalar_declare_with_default(&screen.ay, &screen.ns, "ay", 0);
+    parameter_scalar_declare_with_default(&screen.bx, &screen.ns, "bx", 0);
+    parameter_scalar_declare_with_default(&screen.by, &screen.ns, "by", 0);
+    parameter_scalar_declare_with_default(&screen.cx, &screen.ns, "cx", 0);
```

Comment:
Might be better not to have any defaults as the default value of 0. will cause the screen to act in weird ways. I would prefer to have an assert fail.